docs/content/docs/configuration.md

```diff
@@ -51,6 +54,118 @@ status:
 default_decision: /http/example/authz/allow
 ```
 
+## Environment Variable Substitution
+> Only supported with the OPA runtime (`opa run`).
+
+Environment variables referenced with the `${...}` notation within the configuration
+will be replaced with the value of the environment variable.
+
+Example using `BASE_URL` and `BEARER_TOKEN` environment variables:
+```yaml
+services:
+  - name: acmecorp
+    url: ${BASE_URL}
+    credentials:
+      bearer:
+        token: "${BEARER_TOKEN}"
+
+discovery:
+  name: /example/discovery
+  prefix: configuration
+```
+The environment variables `BASE_URL` and `BEARER_TOKEN` will be substituted in when the config
+file is loaded by the OPA runtime.
+
+> If the variable is undefined then an empty string (`""`) is substituted. It will __not__
+raise an error.*
+
+## CLI Runtime Overrides
+> Only supported with the OPA runtime (`opa run`).
+
+Using `opa run` there are CLI options to explicitly set config values. These will override
+any values set in the config file.
+
+There are two options to use: `--set` and `--set-file`
+
+Both options take options in a `key=value` format where they key is a selector for the yaml
+config structure, for example: `decision_logs.reporting.min_delay_seconds=300` is equivelent
+to JSON `{"decision_logs: {"reporting": {"min_delay_seconds: 300}}}`. Multiple values can be
+specified with comma seperators (`key=value,key2=value2,..`). Or with additonal `--set`
```

Comment:
NIT: To be consistent your the option definition `key1=value1,key2=value2,..`

Reply:
Fixed